### PR TITLE
Boardroom tests update according to new Boardroom.sol

### PR DESCRIPTION
**Pull request by Dobrokhvalov**

### CODE REVIEW

File: contracts/Boardroom.sol [hunk 1]
```diff
@@ -151,9 +165,9 @@ contract Boardroom is ShareWrapper, ContractGuard, Operator {
         directorExists
         updateReward(msg.sender)
     {
-        require(amount > 0, 'Boardroom: Cannot withdraw 0');
-        super.withdraw(amount);
-        emit Withdrawn(msg.sender, amount);
+      require(amount > 0, 'Boardroom: Cannot withdraw 0');
```
Comment (fr-jerry-smith) [inline contracts/Boardroom.sol:168]: We are currently uses four-spaces, so please change this..!

**Review by fr-jerry-smith — APPROVED** [review]
LGTM

**Comment by fr-jerry-smith** [comment]
And tests still not passed with this error
```
Error:  "2020-12-06T17:59:42.584Z"  'error from getOptions'
Error: Error: Command failed: git cat-file -p 948f4f3831a63644371cb0919884030a7d466878
```
I can merge it after fixed

File: contracts/Boardroom.sol (hunk 1 above)
Comment (Dobrokhvalov) [reply contracts/Boardroom.sol]: Fixed

**Comment by Dobrokhvalov** [comment]
Seems like that's some weird random coverrails issue - https://github.com/nickmerwin/node-coveralls/issues/80

Also checks passed in the previous commit ("formatting spaces in Boardroom.sol") - https://github.com/Basis-Cash/basiscash-protocol/runs/1507067340

> And tests still not passed with this error
> 
> ```
> Error:  "2020-12-06T17:59:42.584Z"  'error from getOptions'
> Error: Error: Command failed: git cat-file -p 948f4f3831a63644371cb0919884030a7d466878
> ```
> 
> I can merge it after fixed

**Comment by fr-jerry-smith** [comment]
OK, just merge it. Thank you for supporting us!

**Pull request merged**